monai/data/image_reader.py

```diff
@@ -121,8 +121,11 @@ class ITKReader(ImageReader):
     def __init__(self, **kwargs):
         super().__init__()
         self.kwargs = kwargs
-        if int(itk.Version.GetITKMajorVersion()) == 5 and int(itk.Version.GetITKMinorVersion()) <= 2 \
-                and int(itk.Version.GetITKBuildVersion()) <= 0:
+        if (
+            int(itk.Version.GetITKMajorVersion()) == 5
+            and int(itk.Version.GetITKMinorVersion()) <= 2
+            and int(itk.Version.GetITKBuildVersion()) <= 0
```

Comment:
If so, we don't need this condition.

Reply:
updated in the latest commit, is it expected now?
Thanks.

Reply:
Looks good now.